drivers/flash/nrf_qspi_nor.c

```diff
@@ -1119,6 +1118,8 @@ static int qspi_nor_configure(const struct device *dev)
 
 	ANOMALY_122_UNINIT(dev);
 
+	const struct qspi_nor_config *params = dev->config;
```

Comment:
What about moving it to `qspi_nor_read_id()` where it is actually needed?

Reply:
Done